### DISCUSSION TITLE
Sorry, we had to truncate this directory to 1000 files.289 entrie were omitted from the list

### DISCUSSION THREAD

**Discussion by manigitub** [community/community#21480] [Other Feature Feedback, Questions, & Ideas]
<p>Hi</p>
<p>Some of the files are not committed.</p>
<p>Sorry, we had to truncate this directory to 1000 files.289 entire were omitted from the list.</p>
<p>Kindly help me</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>This is an artificial limitation of Github and only impacts viewing via the browser. If you want to see all the files, you need to clone the repository locally.</p>

**Reply by Amirkia92** [reply]
How can we load them in codespace then?

**Comment by ClaireCJS** [comment]
Wow. This sucks. I really explicitly wanted to list all of these.

**Comment by fmartian** [comment]
One approach I saw at bitbucket would be to have a filter pattern entry at the top that allows to restrict the listing of files to those that match a specific string. It's annoying to know what file (approximately) you want to look at and having to clone the whole repository locally just to see that file.

**Comment by ClaireCJS** [comment]
I ended up making a 2ⁿᵈ folder that is a copy of the first, then deleting [a-p]*.* so that the 1ˢᵗ folder is A-P and the 2ⁿᵈ folder is S-Z. But it’s so tacky. The browser can handle more than 1000 and Ajax loading could lazy-load.
